downloader/src/main/kotlin/Downloader.kt

```diff
@@ -201,11 +201,11 @@ object Downloader {
                         "see: https://guides.rubygems.org/specification-reference/#metadata"
                 "Gradle" -> " Please make sure the released POM file includes the SCM connection, see: " +
                         "https://docs.gradle.org/current/userguide/publishing_maven.html#" +
-                        "example_customizing_the_pom_file"
+                        "sec:modifying_the_generated_pom"
                 "Maven" -> " Please define the \"connection\" tag within the \"scm\" tag in the POM file, " +
                         "see: http://maven.apache.org/pom.html#SCM"
                 "NPM" -> " Please define the \"repository\" in the package.json file, see: " +
-                        "https://docs.npmjs.com/files/package.json#repository"
+                        "https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository"
```

Comment:
It's a but unfortunate that this now links to v7-specific docs, but I also didn't find a way around that while maintaining the anchor.

Reply:
Yes, it's unfortunate that they don't provide an option to link to the current docs like other tools do. But also not too bad, I guess v7 will stay around for a while.